Dockerfile

```diff
@@ -20,22 +20,24 @@
 #  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 #  THE SOFTWARE.
 
+FROM jenkins/agent:latest
 
-FROM jenkins/ssh-agent
+USER root
 
-# install python
+# install pythonipip
```

Comment:
@Loghijiaha did you mean # install python-pip?

Reply:
yes , will change it